pkgs/development/python-modules/astropy-healpix/default.nix

```
@@ -3,25 +3,42 @@
 , fetchPypi
 , numpy
 , astropy
-, astropy-helpers
+, astropy-extension-helpers
+, setuptools_scm
```

The [new name](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python-packages.nix#L7386-L7387) follows PEP-503.

```suggestion
, setuptools-scm
```